lib/rubocop/cop/style/empty_literal.rb

```diff
@@ -35,8 +37,12 @@ def on_send(node)
         end
 
         def autocorrect(node)
+          result = correction(node)
+
+          return unless result
```

Comment:
I'm a bit confused here. How come a node reported as offence not be correctable?